### ISSUE TITLE
Toggle style and accessibility

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/heracles-etd#239]
- Refactor toggle implementation to address accessibility issues. See "Reported by SODA" below.
  - I think SODA's 2nd recommended approach "Use the ARIA group/labelledby technique" might be easiest. The 1st recommended approach "Use a fieldset with legend" may be considered "best practice" since using semantic HTML elements whenever possible is preferred.
- Add left and right padding to both `Yes|No` toggle buttons to better match the design. Approx 15px of left and right padding on both `Yes` and `No` should do it.
- Add `class="text-muted"` to the "Click Yes..." instruction.

<img width="450" height="66" alt="Image" src="https://github.com/user-attachments/assets/3567d2e3-202b-4181-abbf-353fb573d9e3" />
<img width="508" height="89" alt="Image" src="https://github.com/user-attachments/assets/181e0059-8fdf-4c7f-8952-ab55efcdc0e8" />

- Time permitting, it would be great to add the `Yes|No` toggle to the component library: https://github.com/sul-dlss/component-library/issues/183.

### Reported by SODA
> 
> [Summary]
> Group of radio buttons not associated with group label
> 
> [Description]
> The group label is not associated with its group of radio buttons. The purpose of individual radio buttons in the group cannot be understood without the group label.
> A DIV with a group surrounds radio buttons but it is not correctly built. Use a fieldset instead of DIV with group or create the group correctly. 
> 
> [Impact]
> Serious
> 
> [Recommended to fix]
> 
> Use a fieldset with legend: 
> 
> ```
> <fieldset>
>   <legend>Does your dissertation include copyrighted material?<br>Upload additional files to include them in your submission.</legend>
>   <input type="radio" value="true" name="submission[permissions_provided]" id="submission_permissions_provided_true">
>   <label for="submission_permissions_provided_true">Yes</label>
>   <input type="radio" value="false" name="submission[permissions_provided]" id="submission_permissions_provided_false">
>   <label for="submission_permissions_provided_false">No</label>
> </fieldset>
> ```
> 
> Rule : 
> Group labels MUST be programmatically-associated with the group if the individual labels for each element in the group are insufficient on their own.
> 
> How To Fix: 
> Fix this issue by using ONE of the following techniques:
> 1. Use the fieldset/legend technique:
> a. Enclose the group label in a `<legend>` element.
> b. Enclose the `<legend>` element and all of the related form fields in a `<fieldset>` element.
> 
> ```
> <fieldset>
> <legend>Preferred contact method</legend>
> <label for="email">Email</label>
> <input type="radio" id="email" name="contact">
> <label for="phone">Phone</label>
> <input type="radio" id="phone" name="contact">
> </fieldset>
> ```
> 
> 2. Use the ARIA group/labelledby technique:
> a. Enclose the group label in a container such as a `<div>` or a `<p>` and give it a unique id attribute value.
> b. Enclose the related form controls in a container such as a `<div>`.
> c. Apply the `role="group"` and aria-labelledby attributes to the container.
> d. Make the value of the aria-labelledby attribute match the value of the id attribute on the group label container.
> 
> ```
> <p id="contact-label">Preferred contact method</p>
> <div role="group" aria-labelledby="contact-label">
> <label for="email">Email</label>
> <input type="radio" id="email" name="contact">
> <label for="phone">Phone</label>
> <input type="radio" id="phone" name="contact">
> </div>
> ```
> 
> Reference: 
> Deque University: https://dequeuniversity.com/class/forms/group-labels/semantic
> W3C-WAI tutorial: https://www.w3.org/WAI/tutorials/forms/
> 
> Background: 
> People who are blind cannot use the visual layout of a group of related form elements and their shared group label - such as a question with a set of radio button answers - to determine the relationship between the form fields and the group label. In order to understand the relationship of the form fields, their individual labels, and their group label, the relationships must be expressed semantically. When grouped form elements and their group label are expressed semantically, a screen reader user can put focus on a form element and the screen reader will read the form element label and element type as well as the group label.
> 
> Grouping controls is most important for related radio buttons and checkboxes.
> 
>
